Tests fail because of scipy deprecation

### Version Dependencies

Python 3.12.8
Acoular 24.10
macos 14.7.2 (23H311)

### Bug Description

The tests fail because of a deprecation in scipy.

### Reproducible Code

_No response_

### Error message:

```shell
FAILED tests/regression/test_generator.py::test_generators[sources-SphericalHarmonicSource-1ch] - DeprecationWarning: `scipy.special.sph_harm` is deprecated as of SciPy 1.15.0 and will be removed in SciPy 1.17.0. Please use `scipy.special.sph_harm_y` instead.
```


### Issue Severity

None

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -154,6 +154,7 @@ filterwarnings = [
     "ignore:module 'sre_constants' is deprecated:DeprecationWarning", # raised internally by traits
     "ignore:The TBB threading layer requires TBB version:numba.core.errors.NumbaWarning", # raised internally by numba
     "ignore:FNV hashing is not implemented in Numba:UserWarning", # raised internally by numba
+    "ignore:`scipy.special.sph_harm` is deprecated*:DeprecationWarning", # raised by SciPy
     # Suppress warnings about unclosed files in tables at the end of the tests
     "ignore:Closing remaining open file*:tables.exceptions.UnclosedFileWarning",
 ]

